### DISCUSSION TITLE
Refreshing top level element raises warning

### DISCUSSION THREAD

**Discussion by adarshpunj** [zauberzeug/nicegui#3613] [Q&A]
### Question

Suppose I want to refresh `ui.left_drawer` 

```python
@ui.refreshable
def _update_side_nav(chatarea):
    SideNav(chatarea=chatarea).render() # This component contains ui.left_drawer
```
However, doing this raises:
```
Found top level layout element "LeftDrawer" inside element "RefreshableContainer". Top level layout elements should not be nested but must be direct children of the page content. This will be raising an exception in NiceGUI 1.5
```
Is there a better way to achieve this (other than only refereshing an specific section inside `SideNav` which is not a top level element)

**Comment by rodja** [comment]
A `ui.left_drawer` is part of https://nicegui.io/documentation/page_layout#page_layout. These elements will always be placed top level. Otherwise they would not be able to fulfill their purpose. The refreshable should only change the elements inside the drawer, not the drawer itself.
